storage/rocksdb/ha_rocksdb.cc

```diff
@@ -13081,13 +13081,29 @@ static int rocksdb_validate_update_cf_options(
     my_error(ER_WRONG_VALUE_FOR_VAR, MYF(0), "rocksdb_update_cf_options", str);
     return HA_EXIT_FAILURE;
   }
-  // Loop through option_map and create missing column families
+  // Loop through option_map and check if all specified CFs exist.
+  std::vector<const std::string *> unknown_cfs;
   for (Rdb_cf_options::Name_to_config_t::iterator it = option_map.begin();
        it != option_map.end(); ++it) {
-    if (!cf_manager.get_or_create_cf(rdb, it->first,
-                                     !rocksdb_no_create_column_family)) {
-      return HA_EXIT_FAILURE;
+    if (!cf_manager.get_cf(it->first)) {
```

Comment:
Since we're talking 'modern C++' a bit more these days, why not take this opportunity to change this to `for (auto& it : option_map)` paradigm?

Reply:
done